cdk/lib/security-hq.ts

```diff
@@ -118,6 +128,16 @@ dpkg -i /tmp/installer.deb`,
       },
     });
 
+    // TODO reduce TTL and point to new ALB after going live.
+    const oldElb = template.getResource('LoadBalancer') as CfnLoadBalancer;
+
+    new GuCname(this, 'security-hq.gutools.co.uk', {
+      app: 'security-hq',
```

Comment:
Minor, I think we can use this here too?

```suggestion
      app: SecurityHQ.app.app,
```